assets/src/apps/authoring/components/Flowchart/toolbar/FlowchartHeaderNav.tsx

```diff
@@ -172,6 +173,7 @@ export const FlowchartHeaderNav: React.FC<HeaderNavProps> = () => {
     dispatch(setCurrentSelection({ selection: newPartData.id }));
 
     dispatch(setRightPanelActiveTab({ rightPanelActiveTab: RightPanelTabs.COMPONENT }));
+    dispatch(setCopiedPart({ copiedPart: null }));
```

Comment:
Once a copied component is pasted, we need to clear the selection otherwise it always tries to paste the part when you do something else.